### PR TITLE
Proposal for Distributed Workloads

### PR DESCRIPTION
**Pull request by heyselbi**
Proposal for Distributed Workloads and Workload Orchestration

## Description
Adding written proposal for Distributed Workloads and Workload Orchestration

## How Has This Been Tested?
Proposal does not affect other areas of the code, and hence, was not tested.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: sig-platform/distributed-workloads.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+## Architecture
+The architecture and typical workflow for a sample Ray distributed workload are depicted below.
+![Screenshot from 2022-12-02 14-57-05](https://user-images.githubusercontent.com/43946617/206355518-4b3ecd06-7290-4c4c-9f21-8bfb02011927.png)
+
+------------------------------------------------------------------------------------------------------------------------------
+
+![Screenshot from 2022-12-02 14-56-13](https://user-images.githubusercontent.com/43946617/206355725-96eba2dd-1b2f-489e-a396-981bdc6f0819.png)
+
+## Implementation
```
Comment (LaVLaS) [inline sig-platform/distributed-workloads.md:59]: @heyselbi Is this community document going to be the single source of truth for this integration component, if not then this implementation should be stored with the required deployment files for the integration

Comment (heyselbi) [reply sig-platform/distributed-workloads.md]: Removed most of the implementation and left the demo part. I also added the readme file to wg-distributed-workloads directory

File: proposal/distributed-workloads.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+[Multi-Cluster Application Dispatcher](https://github.com/IBM/multi-cluster-app-dispatcher) (MCAD) is a Kubernetes controller providing mechanisms for applications to manage batch jobs in a single or multi-cluster environment. It is capable of (i) providing an abstraction for wrapping all resources of the job or application and treating them holistically, (ii) queuing job or application creation requests and applying different queuing policies, e.g., First In First Out, Priority, and (iii) dispatching the job to one of multiple clusters, where a MCAD queuing agent runs, using configurable dispatch policies.
+
+[InstaScale](https://github.com/IBM/multi-cluster-app-dispatcher/tree/cluster-autoscale) is an on-demand resource scaling tool in the cloud. Within the resource limits set by the user, InstaScale can quickly scale up aggregated resources for distributed computing jobs, and just as easily release the resources once the job (ie. MCAD AppWrapper job) is complete.
```
Comment (LaVLaS) [inline proposal/distributed-workloads.md:15]: Can you update the repo url to `project-codeflare` org and instascale repo?
https://github.com/project-codeflare/instascale

File: proposal/distributed-workloads.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+Atin Sood - IBM Research team - IBM
+
+The owners are committed to maintaining and testing the Distributed Workloads with each new release of its every component and ODH.
+
+## More Info
+CodeFlare SDK: https://github.com/project-codeflare/codeflare-sdk
+
+MCAD: https://github.com/IBM/multi-cluster-app-dispatcher → will be moving under CodeFlare organization Dec 2022
+
+InstaScale: https://github.com/IBM/multi-cluster-app-dispatcher/tree/cluster-autoscale  → will be moving under CodeFlare organization Dec 2022
```
Comment (LaVLaS) [inline proposal/distributed-workloads.md:77]: You can update this now that they have been relocated

File: wg-distributed-workloads/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+* Develop guides, documentation and tutorials for all of the above
+....
+
+## Meetings
+* Status updates will occur during [Platform SIG](sig-platform) meetings
+* [Slack](https://join.slack.com/t/odh-io/shared_invite/zt-18ptx7far-SWO4jkDbuA7Sq8Mut3JbcA) on channel #wg-distributed-workloads
+
+## Organizers
+* Selbi Nuryyeva: [heyselbi](https://github.com/heyselbi)
+* Mustafa Eyceoz: [Jooho](https://github.com/Jooho)
```
Comment (LaVLaS) [inline wg-distributed-workloads/README.md:20]:
```suggestion
* Mustafa Eyceoz: [Maxusmusti](https://github.com/Maxusmusti)
```

**Review by LaVLaS — CHANGES_REQUESTED** [review]

File: wg-distributed-workloads/README.md (hunk 4 above)
Comment (heyselbi) [reply wg-distributed-workloads/README.md]: Updated it and slack link in a commit

File: proposal/distributed-workloads.md (hunk 3 above)
Comment (heyselbi) [reply proposal/distributed-workloads.md]: All links have been updated in a commit

**Review by LaVLaS — APPROVED** [review]
/lgtm

**Pull request merged**